packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx

```diff
@@ -261,16 +322,28 @@ export const usePeoplePane = (props: {
 
   const onRenderContent = useCallback((): JSX.Element => {
     return (
-      <PeoplePaneContent
-        inviteLink={inviteLink}
-        onFetchAvatarPersonaData={onFetchAvatarPersonaData}
-        onFetchParticipantMenuItems={onFetchParticipantMenuItemsForCallComposite}
-        setDrawerMenuItems={setDrawerMenuItems}
-        mobileView={mobileView}
-        setParticipantActioned={setParticipantActioned}
-        participantListHeadingMoreButtonProps={sidePaneHeaderMenuProps}
-        pinnedParticipants={pinnedParticipants}
-      />
+      <>
+        {
+          /* @conditional-compile-remove(soft-mute) */
+          <Prompt
```

Comment:
Should the prompt state and rendering be moved to separate file, looks like we are adding lots of content to this hook `usePeoplePane`

Reply:
It would only move the labels and the prompt component. 
Not too much code would be extracted out. We definitely could if you think it's still necessary.

Reply:
Does the prompt cover only the PeoplePane? I remember I added a prompt for spotlight in the CallPage so that it covers entire call including the VideoGallery, all panes, and control bar.

Reply:
From my testing prompt covers the whole app, not just the peoplepane. I did notice that you added a different prompt higher up the tree for spotlight. Did you attempt to also add a prompt in people pane before you added it up the tree? 
Cause for me prompt works fine

Reply:
I think ideally all small or big components should be kept separately and not inside the hook, but if we think it is not a minor change we can keep it as it is.

Reply:
noted! 
Will revisit this when capabilities is made available.